presto-docs/src/main/sphinx/release/release-0.282.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.282
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix ``TEMPORARY`` definition :doc:`/sql/create-function` and :doc:`/sql/drop-function`.
+* Fix a bug where ``cardinality(map_keys(x))`` and ``cardinality(map_values(x))`` would return wrong results.
+* Fix Presto builds on ARM based processors.
+* Fix Explain (TYPE VALIDATE) to return after the analysis and ACL checks. It no longer executes the dummy query. It now returns ``true`` in the output column ``result`` instead of ``valid`` in case of successful execution.
```

Comment:
This is really an optimization rather than a fix. I would rewrite it as below and move it to the optimization section.

```
Improve performance of ``Explain (TYPE VALIDATE)``` by returning immediately after analysis and ACL checks complete without executing a dummy query. In successful execution, the output column is now called ``result`` rather than ``valid``.
```